src/go/rpk/pkg/cli/cmd/wasm/deploy_test.go

```diff
@@ -0,0 +1,93 @@
+package transform_test
+
+import (
+	"bytes"
+	"strings"
+	"testing"
+	transform "vectorized/pkg/cli/cmd/wasm"
+
+	"github.com/Shopify/sarama"
+	"github.com/Shopify/sarama/mocks"
+	"github.com/sirupsen/logrus"
+	"github.com/spf13/afero"
+	"github.com/stretchr/testify/require"
+)
+
+type FileInformationTest struct {
+	name	string
+	content	string
+}
+
+func TestNewDeployCommand(t *testing.T) {
+	tests := []struct {
+		name		string
+		producer	func(bool, int32) (sarama.SyncProducer, error)
+		fileInformation	FileInformationTest
+		args		[]string
+		expectedOutput	[]string
+		expectedErr	string
+	}{
+		{
+			name:	"it should publish a message with correct format",
+			args:	[]string{"--description", "coprocessor description"},
+			fileInformation: FileInformationTest{
+				name:		"fileName.js",
+				content:	"let s = 'text'",
+			},
+		}, {
+			name:	"it should fail if the file extension isn't js",
+			fileInformation: FileInformationTest{
+				name: "fileName.html",
+			},
+			expectedErr:	"deploy just allow js extension file",
+		},
```

Comment:
What if the file isn't there? What if it can't connect to the broker? Try to come up with more scenarios.

Reply:
Got it